src/index.html

```diff
@@ -4,6 +4,7 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Comic+Neue:wght@400;700&display=swap" rel="stylesheet"/>
     <title>Leaderboard</title>
 </head>
 
```

Comment:
- [ ] Your and _ReadMe_ is **not descriptive** and **incomplete**. Kindly note that _ReadMe_ is mostly the first place that visitors of this project's repository will go and therefore it has to be very detailed and professional. Also, it needs to be done well with respect to this project,  which will is likely to feature on your portfolio. Kindly ensure to improve upon your ReadMe. You may use [this template](https://github.com/microverseinc/readme-template) to craft a professional ReadMe but remember to customize it to your project.